src/staticdata.c

```diff
@@ -3630,7 +3636,7 @@ static int jl_validate_binding_partition(jl_binding_t *b, jl_binding_partition_t
         jl_sym_t *name = b->globalref->name;
         JL_LOCK(&mod->lock);
         jl_atomic_store_release(&mod->export_set_changed_since_require_world, 1);
-        if (mod->usings_backedges) {
+        if (mod->usings_backedges != jl_nothing) {
```

Comment:
This looks like it was a mistake. Can somebody confirm?

Reply:
Yeah, I agree.